postgresql.js

```diff
@@ -247,10 +253,17 @@ module.exports = function (RED) {
 
 								handleDone();
 								downstreamReady = false;
-								node.send(msg);
 								if (tickUpstreamNode) {
 									tickUpstreamNode.receive({ tick: true });
 								}
+								if (done) {
```

Comment:
This section should be moved up to where `node.send(msg);` was, before allowing the reception of a new message (tick).

Reply:
Fixed!
I did have read up on then `done` call here https://nodered.org/blog/2019/09/20/node-done to make sure I did it right. To my understanding the `done` call should be at the very end of the node